### PR TITLE
Chore/update fork

### PR DESCRIPTION
**Pull request by sparta15**
Update fork with changes in original swagger-codegen. This change solves problem with endpoints with more than one id inside the url.

### CODE REVIEW

File: modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java [hunk 1]
```diff
@@ -26,6 +47,55 @@ protected CodegenConfig getCodegenConfig() {
...
+    @AfterMethod
+    public void tearDown() throws Exception {
+        folder.delete();
+    }
+
+    @Test(description = "Test routes generated with more than one ids")
+    public void routeWithMoreIds() {
+        final File output = folder.getRoot();
+
+        Swagger swagger = new SwaggerParser().read("/home/gpastor/Code/baikal-adapter-br/timeline/conf/swagger.json");
```
Comment (palmerabollo) [inline modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java]: - This test only works on your laptop.
- If you think the test is relevant, you should try to contribute it back to the official swagger-codegen repository. This way others can benefit from it and our fork will be easier to maintain.

**Review by palmerabollo — CHANGES_REQUESTED** [review]

File: modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java (hunk 1 above)
Comment (tzimisce012) [reply modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java]: You are just updating the fork with the current version of the original project so they probably have a test for it. I guess the test is not needed anymore

Comment (sparta15) [reply modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java]: Oh, my bad  I think I have changed the file route. Anyway, I think like @tzimisce012 if swagger codegen people are honorable devs they have to had a test for this feature.

**Review by tzimisce012 — APPROVED** [review]
Interesting, they have added a way to generate WSClients...

**Review by palmerabollo — APPROVED** [review]

**Pull request merged**